tests/unittests/benchmark/test_runbenchmarks.py

```diff
@@ -367,19 +302,21 @@ def test_run_benchmark(mocker):
     )
 
     setup_mock.assert_called_once_with()
-    run_mock.assert_called_once_with()
-    upload_metrics_mock.assert_called_once_with(s3_bucket=configs["METRICS_BUCKET"], s3_dir="s3_dir")
+    run_mock.assert_called_once_with(framework="AutoGluon:stable")
+    upload_metrics_mock.assert_called_once_with(
+        s3_bucket=configs["METRICS_BUCKET"], s3_dir="s3_dir/None.None.None.None.None.AutoGluon"
```

Comment:
Maybe we can add more than one `run_kwargs` to make this dir more informative?

Reply:
added `toy` and `dummy` values